### ISSUE TITLE
Migration / Import of Users / Members from WP to pH7CMS

### ISSUE THREAD

**Issue by TGHepburn** [pH7Software/pH7-Social-Dating-CMS#514]
Hello All;

After wandering around the wiki, docs, and database for V15  I'm pretty close to making a practice run of moving users from our WP social site to pH7-social-Dataing CMS.   

I know there is an user import function but it does not import avatars or photos.  Also the passwords are an issue with this method.(?)   Maybe a direct SQL insert would work?  I'm just not clear on the images (directory/sizes)?

If any pro users could give some advice on importing 300 users with associated avatars and pw,  I'd be grateful.

**Comment by TGHepburn** [comment]
Issue #314 gave a bit of help for the Import CSV feature, but I have not got it to work - yet.  Even if it did, I'd be left with matching 300 Avatars to member ID's and importing 300 directories named correctly with 5 versions of the image.  

I have been attempting to insert members into DB but the information is not being passed to the front end.  I populated pH7uw_members and pH7uw_members_info, pH7uw_members_privacy.  It seems that the information from table pHuw_members_info is not being read.

Is there something I can do to trigger the DB to read the inserted information?  Am I missing a table that has to have data added?

![firefox_mC74eRNZ6L](https://user-images.githubusercontent.com/23179549/56398785-b7d51900-6207-11e9-9e64-01dcfd07ff6d.png)
![firefox_J5hxpg9L78](https://user-images.githubusercontent.com/23179549/56398790-bd326380-6207-11e9-9461-fbed98208754.png)

**Comment by TGHepburn** [comment]
I get this when importing csv:

`Service Unavailable

The server is temporarily unable to service your request due to maintenance downtime or capacity problems. Please try again later.

Additionally, a 503 Service Unavailable error was encountered while trying to use an ErrorDocument to handle the request.`

Here is my csv file:
`"Username","Username","FirstName","LastName","Email","description","sex","match_sex","birth_date","address","city","state","zipcode","country","phone","website","socialNetworkSite","height","weight"
"Freddyman","Freddy","Fred","Hammer","example11@example.com","Bla Bla Bla Bla Bla Bla ","male","female","1999-01-01","Main St","Calgary","AB",0,"US","NULL","NULL","NULL","NULL","NULL"
"Freddyboy","Daddy","Thor","Oden","example11@example.com","Nothing to say here","male","female","1999-01-01","RR 4","Wimborne","AB","T4G0M2","CA","NULL","NULL","NULL","NULL","NULL"
`

**Comment by TGHepburn** [comment]
"Username","FirstName","LastName","Email","description","sex","match_sex","birth_date","address","city","state","zipcode","country","phone","website","socialNetworkSite","height","weight","IP"
"Freddyman","Fred","Hammer","example11@example.com","Bla Bla Bla Bla Bla Bla ","male","female","1999-01-01","Main St","Calgary","AB",0,"US","","","","","","00.000.000.00"
"Freddyboy","Thor","Oden","example11@example.com","Nothing to say here","male","female","1999-01-01","RR 4","Wimborne","AB","T4G0M2","CA","","","","","","00.000.000.01"

**Comment by polynamaude** [comment]
@TGHepburn I'd go with a real short and easy answer to one of your question **_"Is there something I can do to trigger the DB to read the inserted information?"_**
Simply take some programming classes in a nearby college or university. If you don't understand how does WordPress reads a database and think that you can "make it read" then there's no answer here that's gonna be of any help. There's one thing about user name and information, have the idea went through your mind that maybe both software don't encode the password in the same way ?

Sorry but maybe you should start by building a test server and learning before trying to operate servers (and end up with problems later-on). It's not because you can install a software that you can run a web service.

**Comment by TGHepburn** [comment]
Solved.

I'll give here a very brief overview of how I was able to import the WP users and their profile images:

1. Create same number of user profiles as needed for import using "Generate Bulk Members"  This will ensure that all tables are populated with the minimum required information
2. Using SQL update the table[s] information `members` and `member_info` watch the limit on character length for file name when renaming imported image files 
3. Create image multiples for the different sizes the cms requires with correct naming convention.  Each group of files will be in a directory named the same as the profile's `username` ; all these directorys upload to `data/system/modules/user/avatar/img`
